boards.txt

```diff
@@ -228,7 +228,11 @@ Generic_nRF51822.build.board=GENERIC
 Generic_nRF51822.build.core=nRF5
 Generic_nRF51822.build.variant=Generic
 Generic_nRF51822.build.variant_system_lib=
+<<<<<<< Updated upstream
+Generic_nRF51822.build.extra_flags=-DNRF51 -DNRF51822
+=======
 Generic_nRF51822.build.extra_flags=-DNRF51
+>>>>>>> Stashed changes
```

Comment:
I would only leave their real number if the pins are actually connected to something else. So even if they are broken out, are they any other components were those pins could be wired to?

There is at least an RGB LED that the user would still want to use, no? Is there anything else in the board?